pyomo/contrib/fbbt/tests/test_fbbt.py

```diff
@@ -0,0 +1,315 @@
+import unittest
+import pyomo.environ as pe
+from pyomo.contrib.fbbt.fbbt import fbbt
+try:
+    import numpy as np
+    numpy_available = True
+except ImportError:
+    numpy_available = False
+
+
+class TestFBBT(unittest.TestCase):
+    def test_add(self):
+        if not numpy_available:
```

Comment:
I think this if statement should be promoted to a `unittest.skipIf` decorator on the test method for consistency with other Pyomo tests